tests/tdscf-1/input.dat

```diff
@@ -0,0 +1,37 @@
+UHF_RPA_cc_pvdz = [0.2445704160468683,
+                   0.2878574429978692,
```

Comment:
all the tests need entering in tests/CM, each test needs its own e.g., tests/tdscf-N/CM, and a quick first line `#! quick description of test` for each would be appreciated.